[2025-02-22T13:01:55.144-0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-22T13:01:55.153-0300] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fii_info.store.store manual__2025-02-22T16:01:48.313637+00:00 [queued]>
[2025-02-22T13:01:55.157-0300] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fii_info.store.store manual__2025-02-22T16:01:48.313637+00:00 [queued]>
[2025-02-22T13:01:55.157-0300] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-22T13:01:55.173-0300] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): store.store> on 2025-02-22 16:01:48.313637+00:00
[2025-02-22T13:01:55.175-0300] {standard_task_runner.py:72} INFO - Started process 18770 to run task
[2025-02-22T13:01:55.179-0300] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'fii_info', 'store.store', 'manual__2025-02-22T16:01:48.313637+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/info_Fii.py', '--cfg-path', '/tmp/tmpzzm5rm8j']
[2025-02-22T13:01:55.180-0300] {standard_task_runner.py:105} INFO - Job 10: Subtask store.store
[2025-02-22T13:01:55.208-0300] {task_command.py:467} INFO - Running <TaskInstance: fii_info.store.store manual__2025-02-22T16:01:48.313637+00:00 [running]> on host LAPTOP-3G01DURD.
[2025-02-22T13:01:55.285-0300] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='fii_info' AIRFLOW_CTX_TASK_ID='store.store' AIRFLOW_CTX_EXECUTION_DATE='2025-02-22T16:01:48.313637+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-22T16:01:48.313637+00:00'
[2025-02-22T13:01:55.286-0300] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-22T13:01:55.289-0300] {taskinstance.py:3313} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 768, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 734, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/decorators/base.py", line 266, in execute
    return_value = super().execute(context)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/lpimenta/pipelineAf/dags/info_Fii.py", line 43, in store_fii
    dados_close.to_csv(caminho,sep=';',decimal=',',index=False,encoding='ANSI')
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/pandas/util/_decorators.py", line 333, in wrapper
    return func(*args, **kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/pandas/core/generic.py", line 3967, in to_csv
    return DataFrameRenderer(formatter).to_csv(
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/pandas/io/formats/format.py", line 1014, in to_csv
    csv_formatter.save()
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/pandas/io/formats/csvs.py", line 251, in save
    with get_handle(
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/pandas/io/common.py", line 723, in get_handle
    codecs.lookup(encoding)
LookupError: unknown encoding: ANSI
[2025-02-22T13:01:55.296-0300] {taskinstance.py:1226} INFO - Marking task as FAILED. dag_id=fii_info, task_id=store.store, run_id=manual__2025-02-22T16:01:48.313637+00:00, execution_date=20250222T160148, start_date=20250222T160155, end_date=20250222T160155
[2025-02-22T13:01:55.315-0300] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-22T13:01:55.315-0300] {standard_task_runner.py:124} ERROR - Failed to execute job 10 for task store.store (unknown encoding: ANSI; 18770)
Traceback (most recent call last):
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3006, in _run_raw_task
    return _run_raw_task(
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 274, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3161, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3185, in _execute_task
    return _execute_task(self, context, task_orig)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 768, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 734, in _execute_callable
    return ExecutionCallableRunner(
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/decorators/base.py", line 266, in execute
    return_value = super().execute(context)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 424, in wrapper
    return func(self, *args, **kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/home/lpimenta/pipelineAf/dags/info_Fii.py", line 43, in store_fii
    dados_close.to_csv(caminho,sep=';',decimal=',',index=False,encoding='ANSI')
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/pandas/util/_decorators.py", line 333, in wrapper
    return func(*args, **kwargs)
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/pandas/core/generic.py", line 3967, in to_csv
    return DataFrameRenderer(formatter).to_csv(
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/pandas/io/formats/format.py", line 1014, in to_csv
    csv_formatter.save()
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/pandas/io/formats/csvs.py", line 251, in save
    with get_handle(
  File "/home/lpimenta/pipelineAf/airflow_env/lib/python3.10/site-packages/pandas/io/common.py", line 723, in get_handle
    codecs.lookup(encoding)
LookupError: unknown encoding: ANSI
[2025-02-22T13:01:55.351-0300] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-02-22T13:01:55.359-0300] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-22T13:01:55.360-0300] {local_task_job_runner.py:245} INFO - ::endgroup::
